src/nix/build.md

```diff
@@ -1,5 +1,7 @@
 R""(
 
+**Note:** this command's interface is based heavily around [*installables*](./nix.md#installables), which you may want to read about first (`nix --help`).
```

Comment:
Cutting&pasting the same message into every command doesn't seem like the right thing to do, and also seems pretty ad hoc (there are a lot of things that these interfaces are based heavily around - e.g. Nix expressions, flakes, the store concept - that we don't refer to here).

It would be better to generically insert a message like "See `nix --help` for information about general concepts."

Reply:
I decided to remove the line altogether, it's visually out of place anywhere I tried to put it and pollutes the search results.